src/RemoteLRS.php

```diff
@@ -265,40 +257,48 @@ private function _parseMultipart($boundary, $content) {
     // as suggested here: http://php.net/manual/en/function.http-parse-headers.php#112986
     //
     // adapted to private method, and force headers to lowercase for easy detection
+    // 
+    // Now reworked so that the variable names make a bit more sense.
```

Comment:
Is all that is changing here the formatting/names of the variables? Does this change serve some other purpose and/or apply for PHP7?